flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/TableSchema.java

```diff
@@ -317,6 +317,20 @@ public Builder field(String name, DataType dataType) {
 			return this;
 		}
 
+		/**
+		 * Add an array of fields with names and data types.
+		 *
+		 * <p>The call order of this method determines the order of fields in the schema.
+		 */
+		public Builder fields(String[] names, DataType[] dataTypes) {
```

Comment:
It appears to me this builder isn't seem necessary as the constructor does exactly the same thing.

Reply:
Note that the builder is already there, and is not introduced by this PR

Reply:
I meant this builder method is new! However, I didn't realize the constructor is private. With that, it's fine to have this.